### PR TITLE
Add Elixir 1.16 and Elixir 1.17 to CI

### PR DESCRIPTION
**Pull request by ypconstante**
Locally some tests are failing with this versions, let's see if on CI they work.
Elixir 1.17 failures are caused by [this change on Elixir](https://github.com/elixir-lang/elixir/commit/8e9cbfcd8c219f9d3558158f1ebee5ec4fadd762) 

I've added:

- [ ] USAGE.md docs if applicable
- [ ] CHANGELOG.md

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -112,6 +112,12 @@ jobs:
           - elixir: 1.15.x
             otp: 25.x
             os: ubuntu-20.04
+          - elixir: 1.16.x
+            otp: 25.x
+            os: ubuntu-20.04
+          - elixir: 1.17.0-rc.1
```
Comment (nathany-copia) [inline .github/workflows/ci.yaml:118]: 1.17.0 is out now

Comment (igas) [inline .github/workflows/ci.yaml:118]:
```suggestion
          - elixir: 1.17.x
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -112,6 +112,12 @@ jobs:
...
+            otp: 25.x
```
Comment (igas) [inline .github/workflows/ci.yaml:119]:
```suggestion
            otp: 27.x
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -112,6 +112,12 @@ jobs:
...
+            os: ubuntu-20.04
```
Comment (igas) [inline .github/workflows/ci.yaml:120]:
```suggestion
            os: ubuntu-22.04
```

**Comment by ypconstante** [comment]
Replaced by https://github.com/elixirs/faker/pull/565

**Pull request closed without merging**